docs/finder/finderspawntab.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Open folders in new tab or new window | Finder
+description: Set whether folders open in a new tab or a new window
+head:
+  - - meta
+    - property: 'og:title'
+      content: macOS defaults > Finder > Open folders in new tab or new window
+  - - meta
+    - property: 'og:description'
+      content: Set whether folders open in a new tab or a new window
+---
+
+# Open folders in new tab or new window
```

Comment:
```suggestion
# Open folders destination
```

As it's also used for the sidebar title, I prefer it to be short and do not assume user prefers one or the other option.